components/04-pages/podcast-pages/podcast-overview-page/_podcast-overview-page.scss

```diff
@@ -0,0 +1,88 @@
+.image-header--podcast {
+  padding: 0;
+}
+
+.podcast-overview-flex {
```

Comment:
ich habe in https://github.com/innoq/innoq-styleguide/compare/podcast-overview-2.0...podcast-overview-2.0-vorschlag einen Vorschlag gemacht, wie ich den Markup anders strukturieren würde (habe es in einem Commit gemacht, anstatt in mehreren Review Commits, weil ich sowohl Markup Anpassungen als auch CSS Anpassungen machen wollte und das leichter in einer Git Diff zu sehen ist.

Meine Hauptänderungen:

* ich habe den `.podcast-overview-flex` in `.podcast-overview-samples` umbenannt -- der Grund dafür ist, dass ich es womöglich vermeide, CSS Properties in den Klassen Namen zu verdrahten. Ich habe lieber eine "fachliche" Name genommen (`samples` ist bestimmt nicht der beste Name, aber ich wollte ein Name, der besagt, dass es sich um eine Menge von Podcasts handelt). Tatsächlich ist `flex` hier eine Implementierungsdetail -- wenn du später Grid für das Layout verwenden wolltest, solltest du das Layout anpassen können, ohne den Namen von der Klasse anpassen zu müssen
* Mit CSS Grid versuche ich immer solche `.grid-item` Verschachtelungen zu vermeiden und versuche meine Kind-Elemente als direkte Kinder in meinem Grid zu platzieren. Deswegen habe ich diese HTML Ebene im Beispiel entfernt.
* Da `.podcast-overview-samples` jetzt ein direktes Kind von meinem Grid ist, kann ich jetzt deklarativ Grid styles an meiner Klasse definieren. Die `span 1` Deklarationen sind bei Grid immer überflüssig, da das der Default-Wert ist. Und dann habe ich überlegt, was man eigentlich auf größere Bildschirme möchte: Dort sollen die Samples immer in der zweite Reihe in meinem Gitter platziert werden. Und dann war `.podcast-overview-samples { grid-row: 2; }` (der genau sagt, dass man den samples immer in der 2 Reihe platzieren sollen) eigentlich ausreichend, da der Grid Layout die Elemente eins nach dem anderen im Grid befüllt, und der Security overview header in der richtige Lücke landet. Für dieses Layout (mit 2 Podcasts) ist das in diesem Fall dann ausreichend. Falls mehrere Podcasts dazukommen würde, müsste man das dann anpassen (wobei man ggf. einfach ein `.nth-of-type` Regel ergänzen müsste)

Reply:
voll gut, danke dir! Hab mir schon gedacht, dass das eleganter geht :)